views/admin/index.html

```diff
@@ -110,6 +110,8 @@ <h3 style="color: orange;">Beta - New Instructor Interface!</h3>
 </div>
 
 <div class="col-md-6">
+
+    {{ if settings.academy_mode: }}
```

Comment:
Can we get the indentation levels right?  The start and ending indentations of this and the pass at the end don't match.  Plus the if is indented but none of the html inside the if is indented at all.  This makes it super confusing to read and follow.

Reply:
pinging @presnick I will accept and get this onto master if you can clean up the issues I identified.